### ISSUE TITLE
massive log when a node is down in rpc

### ISSUE THREAD

**Issue by asias** [scylladb/scylla#981]
In a 3 nodes cluster, when node 2 is killed. 

I saw a lot of rpc errors. This is because in each gossip round, gossip will try to contact unreachable node to check if the node is back. If admin does not do anything, e.g. decommission / removenode, the log error will repeat a lot in each gossip round, once per second. The failure of this gossip syn message attempt is totally fine from the gossip point of view, it is better to log in rpc level: status change on  connectivity per ip:port.
 
```
INFO  2016-03-03 07:54:32,392 [shard 0] storage_service - Starting listening for CQL clients on 127.0.0.1:9042...
INFO  2016-03-03 07:54:49,371 [shard 0] gossip - InetAddress 127.0.0.2 is now DOWN
INFO  2016-03-03 07:54:55,246 [shard 0] rpc - client 127.0.0.2: client connection dropped: Connection refused
INFO  2016-03-03 07:54:56,246 [shard 0] rpc - client 127.0.0.2: client connection dropped: Connection refused
INFO  2016-03-03 07:54:57,246 [shard 0] rpc - client 127.0.0.2: client connection dropped: Connection refused
INFO  2016-03-03 07:54:58,246 [shard 0] rpc - client 127.0.0.2: client connection dropped: Connection refused
INFO  2016-03-03 07:54:59,247 [shard 0] rpc - client 127.0.0.2: client connection dropped: Connection refused
INFO  2016-03-03 07:55:00,247 [shard 0] rpc - client 127.0.0.2: client connection dropped: Connection refused
INFO  2016-03-03 07:55:02,247 [shard 0] rpc - client 127.0.0.2: client connection dropped: Connection refused
INFO  2016-03-03 07:55:04,247 [shard 0] rpc - client 127.0.0.2: client connection dropped: Connection refused
INFO  2016-03-03 07:55:05,247 [shard 0] rpc - client 127.0.0.2: client connection dropped: Connection refused
INFO  2016-03-03 07:55:06,248 [shard 0] rpc - client 127.0.0.2: client connection dropped: Connection refused
INFO  2016-03-03 07:55:07,248 [shard 0] rpc - client 127.0.0.2: client connection dropped: Connection refused
INFO  2016-03-03 07:55:08,248 [shard 0] rpc - client 127.0.0.2: client connection dropped: Connection refused
INFO  2016-03-03 07:55:09,248 [shard 0] rpc - client 127.0.0.2: client connection dropped: Connection refused
INFO  2016-03-03 07:55:10,248 [shard 0] rpc - client 127.0.0.2: client connection dropped: Connection refused
INFO  2016-03-03 07:55:15,249 [shard 0] rpc - client 127.0.0.2: client connection dropped: Connection refused
INFO  2016-03-03 07:55:17,249 [shard 0] rpc - client 127.0.0.2: client connection dropped: Connection refused
INFO  2016-03-03 07:55:20,249 [shard 0] rpc - client 127.0.0.2: client connection dropped: Connection refused
INFO  2016-03-03 07:55:21,249 [shard 0] rpc - client 127.0.0.2: client connection dropped: Connection refused
INFO  2016-03-03 07:55:22,249 [shard 0] rpc - client 127.0.0.2: client connection dropped: Connection refused
INFO  2016-03-03 07:55:24,250 [shard 0] rpc - client 127.0.0.2: client connection dropped: Connection refused
INFO  2016-03-03 07:55:29,250 [shard 0] rpc - client 127.0.0.2: client connection dropped: Connection refused
INFO  2016-03-03 07:55:32,250 [shard 0] rpc - client 127.0.0.2: client connection dropped: Connection refused
INFO  2016-03-03 07:55:33,250 [shard 0] rpc - client 127.0.0.2: client connection dropped: Connection refused
INFO  2016-03-03 07:55:34,251 [shard 0] rpc - client 127.0.0.2: client connection dropped: Connection refused

**Comment by glommer** [comment]
This makes logs gigantic and hard to parse.  Agree that this should be done.
If a node is known to be down we should not keep printing until it is expected to be up again.
